### ISSUE TITLE
Analytics javascript bundle loads twice on exported website

### ISSUE THREAD

**Issue by chuntley** [imbolc/sapper-google-analytics#3]
It seems that on an exported website the analytics javascript bundle loads twice. I'm assuming this is because it loads the SSR version first, then again the client version. Would it make more sense to ignore the SSR version of analytics and only load the bundle on `onMount()`?

**Comment by imbolc** [comment]
Do you mean crating the `script` tag dynamically? SSR version shouldn't send statistic now after #2

**Comment by chuntley** [comment]
Yes, based on the network information that I attached above that it loads the `js?id=UA-` bundle twice. First time initiated from the script tab, the second time from `client...` which looks like the client generated script tag.

**Comment by begonaalvarezd** [comment]
Maybe is related to this issue? https://github.com/sveltejs/sapper/issues/1124

**Comment by chuntley** [comment]
Looks related!
